### ISSUE TITLE
adguard cert does not work with work profile

### ISSUE THREAD

**Issue by terrytw** [AdguardTeam/AdguardForAndroid#4875]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

4.1

### Environment

```markdown
- OS: Android 12
- Device: Xperia 5 II
- Firmware: Latest
```

### Root access

- [X] Yes, I have it.

### Issue Details

The adguard cert seems to only move the last cert to system store. 

This leave a problem, if someone enables work profile, there will be 2 certs, either both in` /data/misc/user/0/cacerts-added`, or in `/data/misc/user/0/cacerts-added` and `/data/misc/user/10/cacerts-added` respectively and adguard only seems to move one of them to the system store. 

### Expected Behavior

Both regular user (0) and work profile user can have certificates in the system store.

### Actual Behavior

For regular user(0) and work profile user, only one of them can have certificate in system store. 

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

**Comment by Versty** [comment]
@terrytw Hi! 
Please check if the issue persists on the latest nightly version. We have made some tweaks to the certificates installation flow recently.

**Comment by terrytw** [comment]
Dear Diana,

Can you please confirm which verison of adguardcert module should I be using in combination with the lastest nightly Adguard Android?

**Comment by Versty** [comment]
@terrytw Yes, please use this version

**Comment by terrytw** [comment]
@Versty Dear Diana, 

I have tried the latest nightly and adguardcert 2.0 beta4, and the problem persists. 

If it helps, you can have the dev try https://play.google.com/store/apps/details?id=com.oasisfeng.island&hl=en_US on your test devices to create work profile and see for yourself. 

The issue is actually quite straight forward, basically:
1. You can not install cert into work profile's user store, you can only install it in the default user's user store;
2. All users share the same system store. 
3. adguardcert module only copies one cert into the system store. 
I have no idea how the intermediate cert schematic works. I hope you guys can figure out a way to make it work. 
For now I manually move cert from user 0's user store to user 10's user store, and use a modified https://github.com/ngorskikh/adguardcert (`cp` instead of `mv`) to address the issue.

**Comment by terrytw** [comment]
@Versty Yes, I know it is intended, which creates a legitimate problem: https filtering does not work in work profile.
